server/routes/payments.js

```diff
@@ -4,9 +4,9 @@ const express = require('express');
 const router = express.Router();
 const PaymentCard = require('../database/models/PaymentCard');
 const isLoggedInGuard = require('../middleware/isLoggedInGuard');
-const ownershipGuard = require('../middleware/ownershipGuard');
+const itemOwnerGuard = require('../middleware/itemOwnerGuard');
```

Comment:
this route will need the cardOwnerGuard. if not created yet, then remove this guard for now